clio-client/src/main/scala/org/broadinstitute/clio/client/dispatch/CommandDispatch.scala

```diff
@@ -19,16 +19,26 @@ class CommandDispatch(val webClient: ClioWebClient, val ioUtil: IoUtil)
     bearerToken: OAuth2BearerToken
   ): Future[HttpResponse] = {
     command match {
-      case GetWgsUbamSchema =>
-        GetWgsUbamSchemaExecutor.execute(webClient, ioUtil)
-      case add: AddWgsUbam =>
-        new AddWgsUbamExecutor(add).execute(webClient, ioUtil)
-      case query: QueryWgsUbam =>
-        new QueryWgsUbamExecutor(query).execute(webClient, ioUtil)
-      case move: MoveWgsUbam =>
-        new MoveWgsUbamExecutor(move).execute(webClient, ioUtil)
-      case delete: DeleteWgsUbam =>
-        new DeleteWgsUbamExecutor(delete).execute(webClient, ioUtil)
+      case GetSchemaWgsUbam =>
+        GetSchemaWgsUbamExecutor.execute(webClient, ioUtil)
```

Comment:
Unfortunately this still puts the differentiator (`WgsUbam`) in the middle of the name `GetSchema...Executor`. Am not sure how far we want to take the rule of putting the name at the start or the end, but it's something to consider here as well.